docs/tutorials/excel-tutorial-create-custom-functions.md

```diff
@@ -3,8 +3,293 @@ title: Excel custom functions tutorial
 description: In this tutorial, you’ll create an Excel add-in that contains a custom function that can perform calculations, request web data, or stream web data.
 ms.date: 12/21/2018
 ms.topic: tutorial
-ROBOTS: NOINDEX
-#Customer intent: As an add-in developer, I want to create a custom function in Excel to increase productivity. 
+#Customer intent: As an add-in developer, I want to create custom functions in Excel to increase user productivity. 
 ---
 
-[!include[Excel custom functions tutorial](../includes/file-tutorial-excel-custom-functions.md)]
\ No newline at end of file
+# Tutorial: Create custom functions in Excel
+
+Custom functions enable you to add new functions to Excel by defining those functions in JavaScript as part of an add-in. Users within Excel can access custom functions just as they would any native function in Excel, such as `SUM()`. You can create custom functions that perform simple tasks such as custom calculations or more complex tasks such as streaming real-time data from the web into a worksheet.
+
+In this tutorial, you will:
+> [!div class="checklist"]
+> * Create a custom functions project by using the Yo Office generator
+> * Use a prebuilt custom function to perform a simple calculation
+> * Create a custom function that requests data from the web
+> * Create a custom function that streams real-time data from the web
+
+[!include[Excel custom functions note](../includes/excel-custom-functions-note.md)]
+
+## Prerequisites
+
+* [Node.js](https://nodejs.org/en/) (version 8.0.0 or later)
+
+* [Git Bash](https://git-scm.com/downloads) (or another Git client)
+
+* The latest version of [Yeoman](https://yeoman.io/) and the [Yeoman generator for Office Add-ins](https://www.npmjs.com/package/generator-office). To install these tools globally, run the following command via the command prompt:
+
+    ```
```

Comment:
Yep, we usually say `bash` for examples like this. But David recently recommended to Michelle that she remove the language for these. Instead of adding it back (going against what David & Michelle decided recently), I'll add this to the agenda for our next team meeting to discuss and decide on which approach we should use throughout the docs.